dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/models/SubSnpCoreFieldsTest.java

```diff
@@ -130,72 +130,97 @@ public void failWithNegativeChromosomeCoordinates() {
                 "NC_003074.8", 23412070L, 23412070L, Orientation.FORWARD,
-                LocusType.SNP, "3", 23412070L, 23412070L,"G", "G",
+                LocusType.SNP, "3", 23412070L, 23412070L, "G", "G",
                 "A", "T/C", "NC_003074.8:g.23412070C>T", 23412070L, 23412070L, Orientation.REVERSE, "NC_003074.8:g.23412070C>T",
                 23412070L, 23412070L, Orientation.REVERSE, "C/T, T/T, C/C, ./., T/C, C, T", "batch");
-        expectedGenotypeList = new ArrayList<>();
-        expectedGenotypeList.add(createGenotypeMap("GT", "0/1"));
-        expectedGenotypeList.add(createGenotypeMap("GT", "1/1"));
-        expectedGenotypeList.add(createGenotypeMap("GT", "0/0"));
-        expectedGenotypeList.add(createGenotypeMap("GT", "./."));
-        expectedGenotypeList.add(createGenotypeMap("GT", "1/0"));
-        expectedGenotypeList.add(createGenotypeMap("GT", "0"));
-        expectedGenotypeList.add(createGenotypeMap("GT", "1"));
-        assertEquals(expectedGenotypeList, subSnpCoreFields.getGenotypeList());
+        List<Map<String, String>> expectedGenotypes = new ArrayList<>();
+        expectedGenotypes.add(createGenotypeMap("GT", "0/1"));
+        expectedGenotypes.add(createGenotypeMap("GT", "1/1"));
+        expectedGenotypes.add(createGenotypeMap("GT", "0/0"));
+        expectedGenotypes.add(createGenotypeMap("GT", "./."));
+        expectedGenotypes.add(createGenotypeMap("GT", "1/0"));
+        expectedGenotypes.add(createGenotypeMap("GT", "0"));
+        expectedGenotypes.add(createGenotypeMap("GT", "1"));
+        assertEquals(expectedGenotypes, subSnpCoreFields.getGenotypes());
+    }
 
+    @Test
+    public void testGenotypes_hyphenated() {
         //Test hyphenated genotypes
-        subSnpCoreFields = new SubSnpCoreFields(492296696, Orientation.REVERSE, 2228714L, Orientation.REVERSE,
+        SubSnpCoreFields subSnpCoreFields = new SubSnpCoreFields(492296696, Orientation.REVERSE, 2228714L, Orientation.REVERSE,
                 "NC_003074.8", 23412070L, 23412070L, Orientation.FORWARD,
-                LocusType.SNP, "3", 23412070L, 23412070L,"G", "G",
+                LocusType.SNP, "3", 23412070L, 23412070L, "G", "G",
                 "", "C/-", "NC_003074.8:g.23412070C>T", 23412070L, 23412070L, Orientation.REVERSE, "NC_003074.8:g.23412070C>T",
                 23412070L, 23412070L, Orientation.REVERSE, "-/ - ", "batch");
-        expectedGenotypeList = new ArrayList<>();
-        expectedGenotypeList.add(createGenotypeMap("GT", "1/1"));
-        assertEquals(expectedGenotypeList, subSnpCoreFields.getGenotypeList());
+        List<Map<String, String>> expectedGenotypes = new ArrayList<>();
+        expectedGenotypes.add(createGenotypeMap("GT", "1/1"));
+        assertEquals(expectedGenotypes, subSnpCoreFields.getGenotypes());
+    }
 
+    @Test
+    public void testGenotypes_MultiAllelic_Reverse() {
```

Comment:
Underscores in names are not among the Java conventions, `testGenotypesMultiallelicReverse` would be more recommended (I am not a fan of camel case myself). Applies to the rest of test methods.